src/authenticationMiddleware.test.ts

```diff
@@ -1,5 +1,11 @@
 import test from "ava";
-import { required } from "./authenticationMiddleware";
+import { audienceRegex, authenticationMiddleware, required } from "./authenticationMiddleware";
+
+const asyncForEach = async (array: string[], callback: Function): Promise<any> => {
```

Comment:
Consider Promise.all instead of having to do a full callback thing.

Reply:
Used a simple for instead of Array.forEach() or Promise.all.